python/gtsam/utils/plot.py

```diff
@@ -119,6 +144,38 @@ def plot_covariance_ellipse_3d(axes,
     axes.plot_surface(x, y, z, alpha=alpha, cmap='hot')
 
 
+def plot_covariance_ellipse_2d(axes,
+                               origin: Point2,
+                               covariance: np.ndarray) -> None:
+    """
+    Plots a Gaussian as an uncertainty ellipse
+
+    The ellipse is scaled in such a way that 95% of drawn samples are inliers.
+    Derivation of the scaling factor is explained at the beginning of this file.
+
+    Args:
+        axes (matplotlib.axes.Axes): Matplotlib axes.
+        origin: The origin in the world frame.
+        covariance: The marginal covariance matrix of the 2D point
+                    which will be represented as an ellipse.
+    """
+
+    w, v = np.linalg.eig(covariance)
```

Comment:
Actually, changing that to [eigh](https://numpy.org/doc/stable/reference/generated/numpy.linalg.eigh.html) is even better I think. This would prevent from getting complex results due to float precision issues. Comment?